rules/PSR4/NodeManipulator/FullyQualifyStmtsAnalyzer.php

```diff
@@ -65,7 +65,7 @@ public function process(array $stmts): void
                 return null;
             }
 
-            return new FullyQualified($name);
+            return new FullyQualified($name, $node->getAttributes());
```

Comment:
better to only map PARENT attribute, as mirror all attributes may cause error on some use cases when checking origNode:

```php
\assert($class === \get_class($origNode));
```

```suggestion
            $fullyQualified = new FullyQualified($name);
            $fullyQualified->setAttribute(Attributekey::PARENT_NODE, $parent);
            
            return $fullyQualified;
```

Reply:
Done.